app/common/auth/login.directive.js

```diff
@@ -61,6 +63,19 @@ function LoginController(
         $scope.failed = false;
         $scope.processing = false;
         $scope.$parent.closeModal();
+        if ($scope.email === 'admin') {
```

Comment:
FYI this `if` block is going out with the bugfixes. just a placeholder

Reply:
Cool. It'd be good to add a comment explain *why* we're redirecting